djangoplicity/pages/tests/test_models.py

```diff
@@ -0,0 +1,150 @@
+from django.conf import settings
+from datetime import datetime, timedelta
+from django.test import TestCase
+from factories import SectionFactory, PageFactory, URLFactory, PageGroupFactory, EmbeddedPageKeyFactory
+from djangoplicity.pages.models import (
+    Page, URL, EmbeddedPageKey, Section,
+    register_page_key, generate_page_id, page_cache_invalidator, 
+    url_cache_invalidator, page_key_cache_invalidator, section_cache_invalidator
+    )
+
+from mock import patch
+
+class SectionTest(TestCase):
+
+    def test_unicode_representation(self):
+        section = SectionFactory.build()
+        self.assertEqual(u'%s' % section, 'TestingSection')
+
+
+class PageTest(TestCase):
+
+    def test_unicode_representation(self):
+        page = PageFactory.build()
+        self.assertEqual(u'%s' % page, 'TestingTitle')
+
+    def test_is_online(self):
+        page = PageFactory.build()
+
+        page.published = False
+        self.assertFalse(page.is_online())
+
+        page.published = True
+        self.assertTrue(page.is_online())
+
+        page.start_publishing = datetime.now() + timedelta(days=1)
+        self.assertFalse(page.is_online())
+        page.start_publishing = None
+
+        page.end_publishing = datetime.now() - timedelta(days=1)
+        self.assertFalse(page.is_online())
+        page.end_publishing = None
```

Comment:
I think we are missing the case when the end_publishing is after today, therefore the is_online should be true in that case.

Reply:
Should be as you said, however this case isn't directly represented in is_online() method, is it?

Reply:
Ok, I think is ok like this already